stage-3--rfc-template.md

```diff
@@ -10,9 +10,12 @@
   of the accepted proposal.
 -->
 
+**If you have feedback, please leave it on the Stage 3 PR (link below).**
+
```

Comment:
Shouldn't this part be put at last?

Reply:
I don't know, I feel like people are more likely to look at the first line than the one below the links. But happy to move it if more maintainers think the same as you!

Reply:
Sounds good, we can wait for someone else to jump in.

Reply:
I think it either being before or after the list is fine